### DISCUSSION TITLE
GitBook survey does not display correctly - cannot fill out the additional feedback field

### DISCUSSION THREAD

**Discussion by JaniceManwiller** [GitbookIO/community#625] [0. General bugs and questions]
I received the GitBook survey email, but after clicking the number for the rating, the additional response field on the next page initially displays as:

![image](https://github.com/GitbookIO/community/assets/107077736/37e22ab4-d3d1-4dfd-94af-ad7b9afc1236)

And when I start typing, it changes to:

![image](https://github.com/GitbookIO/community/assets/107077736/c184cb81-d1c1-4daa-9fc3-6060a2631444)

So there is no way to actually submit the feedback.

I'm on the latest version of Chrome for Mac.

**Comment by aikoii** [comment]
@JaniceManwiller Thanks for flagging! We checked on our end, and it seems to be working fine. Could you please try it again with any/all browser add-ons disabled?

**Comment by JaniceManwiller** [comment]
It actually worked without turning anything off. Must have been a glitch.

**Comment by aikoii** [comment]
@JaniceManwiller Thanks for letting us know! 🤍
